.buildkite/build_windows_installer.sh

```diff
@@ -14,7 +14,7 @@ make writeversion
 cd $KOLIBRI_DOCKER_PATH \
     && git clone https://github.com/learningequality/kolibri-installers.git \
```

Comment:
Hi @aronasorman can you also update this github repo link into `https://github.com/learningequality/kolibri-installer-windows.git`